CS4850Group5A/VirtuDoc
# Favicon

# Summary
- Added favicon in static directory.
- Added manifest for advanced favicon types for mobile browsers.
- Added tests to verify favicon file states.
- Changed `SecurityConfiguration.java` to allow for certain assets from the static asset directory root.

Closes #195

# How to Test
Run the code locally and verify that the website logo appears in your browser's tab for the site (e.g. like how you see the GitHub logo in your browser tab for this page).

# Comments
If you're wondering why these images need to go into the root of the `static/` directory, and not in the `img/`, it's because the W3C standard on favicons say that favicons need to be accessible from the URL root (e.g. `example.com/favicon.ico`).
